### PR TITLE
Feature/new product

### PR DESCRIPTION
**Pull request by Asergey91**
adds a function to create an empty remplate
frontend pr: https://github.com/lblod/frontend-loket/pull/143

### CODE REVIEW

File: lib/createForm.js [hunk 1]
```diff
@@ -4,6 +4,30 @@ import { v4 as uuid } from 'uuid';
 import { APPLICATION_GRAPH, FORM_STATUS_CONCEPT, PREFIXES } from '../config';
 import { bindingsToNT } from '../utils/bindingsToNT';
 
+export async function createEmptyForm() {
+  const publicServiceId=uuid();
+  const publicServiceUri=`<http://data.lblod.info/id/public-service/${publicServiceId}>`
+
+  const now=new Date().toISOString();
+  const query=`
+    INSERT DATA {
```
Comment (cecemel) [inline lib/createForm.js:13]: Is it possible to use the prefixes?

**Review by cecemel — CHANGES_REQUESTED** [review]
thanks

File: lib/createForm.js (hunk 1 above)
Comment (Asergey91) [reply lib/createForm.js:13]: tried it but it caused some errors (see mu-semtech chat)...

Comment (Asergey91) [reply lib/createForm.js]: the problem was that \t s were in the prefixes.... why did everything else work then.... im big confused... well whatever it works now

File: lib/createForm.js [hunk 2]
```diff
@@ -4,6 +4,31 @@ import { v4 as uuid } from 'uuid';
...
+export async function createEmptyForm() {
+  const publicServiceId = uuid();
+  const publicServiceUri = `http://data.lblod.info/id/public-service/${publicServiceId}`
+
+  const now = new Date().toISOString();
+  const query = `
+  ${PREFIXES}
+  INSERT DATA {
+    GRAPH <http://mu.semte.ch/graphs/application> {
+      ${sparqlEscapeUri(APPLICATION_GRAPH)} a cpsv:PublicService ;
```
Comment (cecemel) [inline lib/createForm.js:16]: @Asergey91 this is a bug

**Review by cecemel — CHANGES_REQUESTED** [review]

File: lib/createForm.js (hunk 2 above)
Comment (Asergey91) [reply lib/createForm.js:16]: oh god dammmit ctrlz too much

**Pull request merged**